source/draft_users.rst

```diff
@@ -278,9 +278,16 @@ Delete a user. Returns a 200 OK with empty response body on success, or an
 Role Management
 ---------------
 
-Role management is handled through the ``projects`` endpoints. The projects
-model contains a ``users`` object, which contains three lists: ``members``,
-``spectators``, and ``managers``, each lists of usernames. An admin or project
-manager may set these at any time, adding to or removing from any of the lists.
-A project must always have at least one manager, however. Attempting to remove
-all managers from a project will return an error.
+Role management is handled through the ``projects`` and ``users`` endpoints.
+
+The users model contains the ``site_spectator``, ``site_manager``, and ``site_admin``
```

Comment:
We don't really talk about models in the API docs - since a model is an implementation detail - we can probably talk about a user object, since we can think of the json representation of a user as an object, but we should be careful to talk about the API in purely abstract terms. 

So maybe talk about the site as an abstract object that has managers and spectators, and a project as another kind of object that has managers, members and spectators. It's a little awkward because we don't have a 'site' endpoint with a list of users like we do with projects, but conceptually, as far as the end user knows, such a construct in fact exists.